decidim-bulletin_board-app/app/javascript/packs/sandbox/vote.js

```diff
@@ -127,3 +143,7 @@ $(async () => {
     },
   });
 });
+
+const setPerformedAction = (action) => {
+  performedAction = action;
+};
```

Comment:
Just to avoid polluting the module global space I would move this inside the `$(async() => {})` block. Same for the variables 👍